### PR TITLE
Acceptance test for group names with emoji

### PR DESCRIPTION
**Pull request by bhawanaprasain**
Using these tests we can check if we can use group names with emojis .

Part of issue #34437

### CODE REVIEW

File: tests/acceptance/features/apiProvisioningGroups-v1/deleteGroup.feature [hunk 1]
```diff
@@ -44,6 +44,8 @@ Feature: delete groups
       | 50%2Eagle           | %2E literal looks like an escaped "."   |
       | 50%2Fix             | %2F literal looks like an escaped slash |
       | staff?group         | Question mark                           |
+      | 😁 😂               | emoji                                   |
+
```
Comment (dpakach) [inline tests/acceptance/features/apiProvisioningGroups-v1/deleteGroup.feature]: extra line

File: tests/acceptance/features/apiProvisioningGroups-v2/removeFromGroup.feature [hunk 2]
```diff
@@ -22,7 +22,7 @@ Feature: remove a user from a group
       | España      | special European characters |
       | नेपाली      | Unicode group name          |
 
-  Scenario Outline: admin removes a user from a group
+   Scenario Outline: admin removes a user from a group
```
Comment (dpakach) [inline tests/acceptance/features/apiProvisioningGroups-v2/removeFromGroup.feature]: incorrectly indented

**Comment by individual-it** [comment]
restarted build

**Comment by bhawanaprasain** [comment]
@dpakach @paurakhsharma @skshetry please review again

**Review by paurakhsharma — COMMENTED** [review]
Looks good to me :+1:

**Review by individual-it — APPROVED** [review]

**Comment by individual-it** [comment]
@bhawanaprasain please backport to stable10 branch

**Comment by phil-davis** [comment]
Backport `stable10` #35300

**Pull request merged**